nekoyume/Assets/_Scripts/State/ReactiveShopState.cs

```diff
@@ -92,6 +102,8 @@ public static void Initialize(ShopState state, IEnumerable<ShopItem> shardedProd
             // ItemSubTypeProducts.
             {
                 var agentAddress = States.Instance.AgentState.address;
+
+                // 내 아바타아이디와 같지 않은 것들
```

Comment:
Can you explain more details as English?

Reply:
This is an unnecessary comment. So I deleted